helm-charts/common/speecht5/templates/tests/test-pod.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "speecht5.fullname" . }}-testpod"
+  labels:
+    {{- include "speecht5.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+    #"helm.sh/hook-delete-policy": "hook-succeeded, hook-failure"
+spec:
+  containers:
+    - name: curl
+      image: alpine/curl
+      #image: python:3.10.14
+      command: ['sh', '-c']
+      args:
+        - |
+          curl http://{{ include "speecht5.fullname" . }}:{{ .Values.service.port }}/v1/tts \
```

Comment:
please add curl parameter `-sS --fail-with-body` to make sure curl will return error in case of HTTP status code >400